src/box/memtx_engine.cc

```diff
@@ -88,7 +88,14 @@ static inline struct tuple *
 memtx_tuple_new_raw_impl(struct tuple_format *format, const char *data,
 			 const char *end, bool validate);
 
-template <class ALLOC>
+/**
+ * Run one iteration of garbage collection. Set @stop if
+ * there is no more objects to free.
+ */
+static void
+memtx_engine_run_gc(struct memtx_engine *memtx, bool *stop);
```

Comment:
Let's leave the comment where it is now.

Reply:
Fixed.

```diff
--- a/src/box/memtx_engine.cc
+++ b/src/box/memtx_engine.cc
@@ -88,10 +88,6 @@ static inline struct tuple *
 memtx_tuple_new_raw_impl(struct tuple_format *format, const char *data,
                         const char *end, bool validate);

-/**
- * Run one iteration of garbage collection. Set @stop if
- * there is no more objects to free.
- */
 static void
 memtx_engine_run_gc(struct memtx_engine *memtx, bool *stop);

@@ -1426,6 +1422,10 @@ static const struct engine_vtab memtx_engine_vtab = {
        /* .check_space_def = */ generic_engine_check_space_def,
 };

+/**
+ * Run one iteration of garbage collection. Set @stop if
+ * there is no more objects to free.
+ */
 static void
 memtx_engine_run_gc(struct memtx_engine *memtx, bool *stop)
 {
```